sections/related-works.tex

```diff
@@ -0,0 +1,12 @@
+https://www.investopedia.com/terms/a/autoregressive-integrated-moving-average-arima.asp
+https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.462.3756&rep=rep1&type=pdf
+
+Auto regressive integrated moving average (ARIMA) is a linear statistical analysis model that uses time series data to predict future trends. It comprises two models; The Auto-regression (AR) model which is a model that forecasts a variable using a linear combination of past values of the same variable. The moving average model (MA), which instead of using past values, uses previously forecasted errors (residuals) to make current predictions. 
+The final element in the ARIMA model, the I is the differencing of observations in order to achieve stationarity, which is done to remove trends or seasonality, which will affect the value of the time series at different times. The primary drawback of the ARIMA model is the assumption of linearity in the time series data, which may in fact be non-linear. (https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.462.3756&rep=rep1&type=pdf)
+(https://otexts.com/fpp2/MA.html)
+
+// this is just an idea
+Another approach to time series forecasting is the use of neural networks. A popular model is to use recurrent neural networks and derivations of this model. Recurrent neural networks are similar to feedforward networks, but it has a connection pointing backwards and can be used to generate sequences, or to label or classify sequences. The backwards connection acts as a memory in the network, as it preserves some of the state across time, which makes it capable of learning short patterns. A primary issue with RNN’s are the vanishing gradient and the exploding gradient problem, which more powerful models seek to improve upon. (Artificial intelligence: A modern approach, 4th edition)(Hands on Machine learning with scikit-learn, Keras and TensorFlow, 2nd edition)
```

Comment:
Could we be more precise when describing RNNs? I'm not sure if calling the recurrence function and hidden state a 'connection' is a bit too reductive.